test/instrumentation/decode_metric.go

```diff
@@ -453,25 +454,78 @@ func (c *metricDecoder) decodeInt64(expr ast.Expr) (int64, error) {
 		}
 		return value, nil
 	case *ast.SelectorExpr:
+		println("SelectorExpr")
```

Comment:
It is for debugging or logging? If it is for logging, do we need print for all cases like the `ast.Ident` one?